src/views/v2/month/calendar-body/day.php

```diff
@@ -53,7 +53,7 @@
  * @since 5.16.2
  */
 if ( $day[ 'month_number' ] < date( 'm', strtotime( $today_date ) ) ) {
-	$day_classes[] = 'tribe-events-calendar-month__day--past-month';
+	$day_classes[] = 'tribe-events-calendar-month__day--past-month tribe-events-othermonth';
```

Comment:
```php
/**
 * Add a unique CSS class to any date that is not in the current month.
 *
 * @since TBD
 */
if ( $day[ 'month_number' ] !== date( 'm', strtotime( $today_date ) ) ) {
	$day_classes[] = 'tribe-events-calendar-month__day--other-month';
}
```

and you only have to do it once.

Reply:
This is meant as a separate, standalone conditional instead of mixing ti with the future/past ones.

Reply:
Ooh yeah this works as well. I thought my initial approach or reusing the existing conditionals would be more efficient 🙂